linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_pwhistory_remember_system_auth/tests/correct_value_multiple_control_required.pass.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# packages = pam
+# platform = Oracle Linux 8
+# profiles = xccdf_org.ssgproject.content_profile_stig
```

Comment:
For this test scenario, I would recommend to define the variable value to be testes instead of the profile where it is set. This way we make sure the test scenario is not impacted by any change in the profile. For example:
`variables = var_password_pam_remember=5,var_password_pam_remember_control_flag=ol8`

Reply:
I wanted to do something similar, but it doesn't work in this way, if I set `var_password_pam_remember_control_flag=ol8` the value is literally "ol8" it doesn't take the value from the `.var` file